static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -71,32 +126,49 @@ export function InviteBanner({missingMembers, onSendInvite, organization}: Props
   cards.push(<SeeMoreCard key="see-more" missingUsers={users} />);
 
   return (
-    <StyledCard data-test-id="invite-banner">
-      <CardTitleContainer>
-        <CardTitleContent>
-          <CardTitle>{t('Bring your full GitHub team on board in Sentry')}</CardTitle>
-          <Subtitle>
-            {tct('[missingMemberCount] missing members that are active in your GitHub', {
-              missingMemberCount: users.length,
-            })}
-            <Tooltip title="Based on the last 30 days of commit data">
-              <IconInfo size="xs" />
-            </Tooltip>
-          </Subtitle>
-        </CardTitleContent>
-        <ButtonContainer>
-          <Button
-            priority="primary"
-            size="xs"
-            // TODO(cathy): open up invite modal
-            data-test-id="view-all-missing-members"
-          >
-            {t('View All')}
-          </Button>
-        </ButtonContainer>
-      </CardTitleContainer>
-      <MemberCardsContainer>{cards}</MemberCardsContainer>
-    </StyledCard>
+    <Feature organization={organization} features={['integrations-gh-invite']}>
+      <StyledCard data-test-id="invite-banner">
+        <CardTitleContainer>
+          <CardTitleContent>
+            <CardTitle>{t('Bring your full GitHub team on board in Sentry')}</CardTitle>
+            <Subtitle>
+              {tct(
+                '[missingMemberCount] missing members that are active in your GitHub',
+                {
+                  missingMemberCount: users.length,
+                }
+              )}
+              <Tooltip title="Based on the last 30 days of commit data">
+                <IconInfo size="xs" />
+              </Tooltip>
+            </Subtitle>
+          </CardTitleContent>
+          <ButtonContainer>
+            <Button
+              priority="primary"
+              size="xs"
+              // TODO(cathy): open up invite modal
+              data-test-id="view-all-missing-members"
```

Comment:
do you have t ouse `data-test-id` here? you should be able to use `getByRole('button', {name: 'View All'})`

Reply:
i haven't tested for this yet so can remove it